### DISCUSSION TITLE
5월 26일 오리와 페어

### DISCUSSION THREAD

**Discussion by giscus** [Cyma-s/Cyma-s.github.io#58] [General]
# 5월 26일 오리와 페어

앞으로의 계획 크레딧을 가지고 전체 배포와 인프라 쪽을 옮겨오는 작업 CI/CD 안 되어 있음. 기존에는 손배포 진행 -> jar 로컬에서 만들어서 scp 후 run CI는 github action 으로 되고 있음. CD도 github action을…

https://vero.wiki/ori-05-26/

**Comment by kevstevie** [comment]
아니 개어렵네
